src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js

```diff
@@ -15,7 +16,7 @@ const ReviewerList = ({ reviewers, icon, tooltip }) => {
         <OverlayTrigger placement="left" overlay={tooltipOverlay}>
           <i
             style={{ marginRight: '12px', width: '12px', marginBottom: '6px' }}
-            className={`fa ${icon}`}
+            className={`fa ${icon} ${approval}`}
```

Comment:
There are many of attempts to use constants as part of css class all over the components, but I think from now on we would rather avoid this and add function to return appropriate css class for appropriate state. 

When we get to css modules support it would be very easy to test and catch issues :)